vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1448,14 +1448,18 @@ public void onUnexpectedError(final Exception e) {
      */
     private void joinARoom() {
         LayoutInflater inflater = LayoutInflater.from(this);
-
-        AlertDialog.Builder alertDialogBuilder = new AlertDialog.Builder(this);
         View dialogView = inflater.inflate(R.layout.dialog_join_room_by_id, null);
-        alertDialogBuilder.setView(dialogView);
 
         final EditText textInput = dialogView.findViewById(R.id.join_room_edit_text);
         textInput.setTextColor(ThemeUtils.INSTANCE.getColor(this, R.attr.riot_primary_text_color));
 
+        AlertDialog.Builder alertDialogBuilder = new AlertDialog.Builder(this);
```

Comment:
So I think the alertDialogBuilder variable is not necessary anymore. Can you remove it please?

Reply:
It is actually used later to add some behavior to a button, same for all the other cases.